src/lambdaisland/deep_diff.cljc

```diff
@@ -2,7 +2,7 @@
   (:require [lambdaisland.deep-diff.diff :as diff]
             [lambdaisland.deep-diff.printer :as printer]))
 
-(defn diff
+(defn call-diff
```

Comment:
This is public API, we definitely can't change this. Better to rename the `diff` namespace to `diffing`, same for `printer` -> `printing`